server/handlers/commentHandler.ts

```diff
@@ -0,0 +1,42 @@
+import { CreateCommentRequest, CreateCommentResponse, DeleteCommentRequest, DeleteCommentResponse } from '../api';
+import { db } from '../datastore';
+import { ExpressHandler, Comment } from '../types';
+import crypto from 'crypto';
+
+//Create Comment Handler
+export const createCommentHandler
+    :ExpressHandler<CreateCommentRequest,CreateCommentResponse> = async (req,res)=>{
+
+    if(req.body.postId === '' || req.body.postId === undefined)
```

Comment:
you can just use `if (!req.body.postId)`, that tests both of these. same below.

Reply:
Thanks